messages/env.compute.collaborator.add.md

```diff
@@ -0,0 +1,13 @@
+# summary
+
+Add heroku user to all compute environments.
+
+# description
+
+You must provide the '--heroku-user' required flag.
+
+# examples
+
+- Add a heroku user to all compute environments:
```

Comment:
```suggestion
- Add a Heroku user to all compute environments:
```

Reply:
Also, you need a summary for the `--heroku-user` flag.  Something like (please rewrite to be technically accurate):

----start----
# flags.heroku-user.summary

Email address of the Heroku user you're adding as a collaborator. 

----end----

Then update the add.ts file to pick up this message.